UI/CensusBrowseWnd.cpp

```diff
@@ -216,6 +238,31 @@ void CensusBrowseWnd::CompleteConstruction() {
 
     m_tags_list->Resize(GG::Pt(BrowseTextWidth(), top2 -top -ROW_HEIGHT - HALF_HEIGHT + (EDGE_PAD*3)));
 
+    top2 += 0.5 * ROW_HEIGHT;
+    
```

Comment:
in the editor I see 4 spaces here, though I can't tell if they are here in the web view. if so, please make it a blank line instead

Reply:
and done

Reply:
they were not visible here in the web view for me, i made it a blank line anyways.